packages/phpstan-rules/src/Rules/ForbiddenFuncCallRule.php

```diff
@@ -55,7 +55,8 @@ public function process(Node $node, Scope $scope): array
             return [];
         }
 
-        if (! $this->arrayStringAndFnMatcher->isMatch($funcName, $this->forbiddenFunctions)) {
+        $forbiddenFunctions = array_keys($this->getForbiddenFunctionsWithMessages());
```

Comment:
This only moves the problem to a different line.

Could you extrat it to a new method as suggested?

Reply:
ohh it seems github lags a bit.. I had this line in the PR 10 minutes ago ;-).

Reply:
Ah, I see :)  thanks :+1: